composer.json

```diff
@@ -56,9 +56,9 @@
     "oscarotero/env": "^2.1",
     "roots/bedrock-autoloader": "^1.0",
     "roots/bedrock-disallow-indexing": "^2.0",
-    "roots/wordpress": "6.0.3",
-    "roots/wp-config": "1.0.0",
-    "roots/wp-password-bcrypt": "1.1.0",
+    "roots/wordpress": "*",
```

Comment:
I'm not comfortable with "*" as a pretty messed up composer conflict could even downgrade the site instead of upgrading it.

Could you please put "^" in front of current version? If we ever need a major version change there in the future, hopefully at that time we'll have start states decoupled and we'll be able to do so.